packages/gatsby/index.d.ts

```diff
@@ -1658,7 +1658,7 @@ export interface GatsbyFunctionRequest extends IncomingMessage {
    * Object of values from route parameters
    */
   params: Record<string, string>
-  body: any
+  body: Partial<Body>
```

Comment:
What was your reasoning for using `Partial` here? Isn't that too opinionated and it should only be `Body` so that if someone wants to use `Partial` they can pass it in themselves?

Reply:
Thanks for the link. Renamed to follow express. 
Docs updated!